docs/server/source/simple-network-setup.md

```diff
@@ -240,6 +244,45 @@ Note: We'll share some example startup scripts in the future. This document is a
 
 If you followed the recommended approach and created startup scripts for BigchainDB and Tendermint, then you can reboot the machine now. MongoDB, BigchainDB and Tendermint should all start.
 
+
+### Member: Start BigchainDB and Tendermint using Monit
+
+This section describes how to manage the BigchainDB and Tendermint processes using [Monit][monit] - a small open-source utility for managing and monitoring Unix processes.
+
+This section assumes that you followed the guide down to the [start MongoDB section](member-start-mongodb) inclusive.
```

Comment:
The hyperlink on this line isn't working..

Reply:
Forgot to test it, thank you for spotting it!

Reply:
:+1: